zebra/rt_netlink.c

```diff
@@ -2072,14 +2078,13 @@ static int netlink_nexthop(int cmd, struct zebra_dplane_ctx *ctx)
 				}
 			}
 
-		nexthop_done:
-			if (IS_ZEBRA_DEBUG_KERNEL) {
-				char buf[NEXTHOP_STRLEN];
+nexthop_done:
+
+			if (IS_ZEBRA_DEBUG_KERNEL)
+				zlog_debug("%s: ID (%u): %pNHv (%u) %s ",
```

Comment:
I see we are trying to print vrf's name we well with further commits. Can we use vrf_id_to_name here as well?